### ISSUE TITLE
App crashes on start

### ISSUE THREAD

**Issue by Kazuto** [TablePlus/TablePlus#2778]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
None, crash happens at start.

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
4.8.2 (436)

3. The steps to reproduce this issue:
Don't know. TablePlus crashes on start since this morning.

Here's the [Problem Report](https://pastebin.com/rVj8mr5x) by macOS

**Comment by huyphams** [comment]
Hi @Kazuto, thank you for letting me know. Does this happen every time?

**Comment by Kazuto** [comment]
Yes, it happened every time. I've had to reinstall macOS completely in order to use TablePlus again. Probably some settings of TablePlus got corrupted with no way of resetting the settings. But this is just a guess.

**Comment by huyphams** [comment]
It could be a reason, you can always reset any macOS (not only TablePlus) app settings. For example to reset settings for TablePlus:
You can remove this file: 

```
rm ~/Library/Preferences/com.tinyapp.TablePlus.plist
```

**Comment by nathanjansen** [comment]
I am having the exact same thing. I removed ...TablePlus.plist, but it did not help. Is there any more info on this?

**Comment by huyphams** [comment]
Hi @nathanjansen do you have the crash report? can I see it. Thanks

**Comment by nathanjansen** [comment]
> Hi @nathanjansen do you have the crash report? can I see it. Thanks

I already figured it out with the help of Nick from TablePlus. It seems I used Mackup (almost a year ago) to backup my old mac to my new one. With that it somehow created aliases of ~/Library/Application\ Support/com.tinyapp.TablePlus/data and ~/Library/Application\ Support/com.tinyapp.TablePlus/plugins. Those aliases got corrupted somehow, but the original locations of the aliases where still active. So I was able to copy and paste the folders.

Hope this helps someone :)
